test-integration/scripts/24-tests-e2e.sh

```diff
@@ -43,7 +43,7 @@ launchCurlOrProtractor() {
         return 1
     fi
 
-    if [ "$JHI_PROTRACTOR" != 1 ]; then
+    if [ "$JHI_E2E" != 1 ]; then
```

Comment:
This will disable daily builds tests and other repositories that uses this script.
I think we should test both JHI_PROTRACTOR and JHI_E2E.

Reply:
Exact, we should migrate daily build to use JHI_E2E too.

Reply:
Do not forget that there are used also by jhipster/jhipster project

Reply:
Thanks Daniel, I will migrate all tomorrow and open PRs.